Backend/Remora.Discord.API/API/Objects/Interactions/ApplicationCommands/ApplicationCommand.cs

```diff
@@ -45,5 +45,7 @@ public record ApplicationCommand
     Optional<IReadOnlyDictionary<string, string>?> NameLocalizations = default,
     Optional<string> NameLocalized = default,
     Optional<IReadOnlyDictionary<string, string>?> DescriptionLocalizations = default,
-    Optional<string> DescriptionLocalized = default
+    Optional<string> DescriptionLocalized = default,
+    Optional<bool> DmPermission = default,
```

Comment:
Same here, nullability. The property should also be named DMPermission, since DM is an abbreviation. Try to keep the order of the properties the same as their declaration order, too.

Reply:
> Try to keep the order of the properties the same as their declaration order, too.
Oops, will do